pac4j-oauth/src/main/java/org/pac4j/oauth/profile/generic/DefaultGenericAttributesDefinition.java

```diff
@@ -0,0 +1,35 @@
+package org.pac4j.oauth.profile.generic;
+
+import org.pac4j.core.profile.AttributesDefinition;
+import org.pac4j.core.profile.converter.Converters;
+
+/**
+ * Default attribute definition of the GenericOAuth20Profile
+ *
+ * @author aherrick
+ * @since 1.9.2
+ */
+public class DefaultGenericAttributesDefinition extends AttributesDefinition {
```

Comment:
This definition (renamed as `CommonAttributesDefinition`) should be put in the core as it's primarily attached to the `CommonProfile`.